libpod/container_linux.go

```diff
@@ -0,0 +1,34 @@
+// +build linux
+
+package libpod
+
+import (
+	"github.com/containernetworking/plugins/pkg/ns"
+	"github.com/projectatomic/libpod/pkg/rootless"
+)
+
+type containerStateLinux struct {
+
+	// NetNSPath is the path of the container's network namespace
+	// Will only be set if config.CreateNetNS is true, or the container was
+	// told to join another container's network namespace
+	NetNS ns.NetNS `json:"-"`
+}
+
+// containerStateWindows is intentionally left as a blank stub
+type containerStateWindows struct {
+}
+
+// containerStateDarwin is intentionally left as a blank stub
+type containerStateDarwin struct {
+}
+
+// IsRootless returns a bool if the container is rootless
+func IsRootless() bool {
```

Comment:
We should just make the rootless package conditional-compile instead of stubbing it out here

Reply:
done